common/include/common/common_utils.hpp

```diff
@@ -290,6 +293,44 @@ static inline std::vector<blas2_param_t<scalar_t>> get_blas2_params(
   }
 }
 
+/**
+ * @fn get_copy_params
+ * @brief Returns a vector containing the blas1 copy benchmark parameters,
+ * either read from a file according to the command-line args, or the default
+ * ones.
+ */
+template <typename scalar_t>
+static inline std::vector<copy_param_t<scalar_t>> get_copy_params(Args& args) {
+  if (args.csv_param.empty()) {
+    warning_no_csv();
+    std::vector<copy_param_t<scalar_t>> default_values;
+    for (index_t incx = 1; incx <= 8; incx *= 2) {
```

Comment:
For the sake of benchmarks we have been trying to keep the `inc*` param values fixed to `1`. So I think we can do the same here as well.